--- pwsafe-0.99BETA.orig/src/core/CMakeLists.txt
+++ pwsafe-0.99BETA/src/core/CMakeLists.txt
@@ -97,4 +97,4 @@
   list (APPEND CORE_SRCS core.rc2)
 endif(NOT WIN32 OR WX_WINDOWS)
 
-add_library(core ${CORE_SRCS})
+add_library(core STATIC ${CORE_SRCS})
--- pwsafe-0.99BETA.orig/src/os/CMakeLists.txt
+++ pwsafe-0.99BETA/src/os/CMakeLists.txt
@@ -84,4 +84,4 @@
     list(APPEND OS_SRCS ${OS_YUBI_SRC})
 endif()
 
-add_library(os ${OS_SRCS})
+add_library(os STATIC ${OS_SRCS})
